### ISSUE TITLE
send-comment error on example project using BitBucket

### ISSUE THREAD

**Issue by pdmct** [iterative/cml#961]
Hi,

New to cml.
I am trying to setup the example project in a bitbucket repo. 
( needed to make a few minor changes to the example in the doco to get it running)
bitbucket-pipelines.yml
```
image: iterativeai/cml:0-dvc2-base1
pipelines:
  default:
    - step:
        name: Train model
        script:
          - pip install -r requirements.txt
          - python train.py
        artifacts:
          - metrics.txt
          - plot.png
    - step:
        name: Create CML report
        script:
          - cat metrics.txt > report.md
          - echo >> report.md
          - cml publish plot.png --md >> report.md
          - cml send-comment report.md
```

I have the pipeline running up to the send-comment command where I get the following error:
```
cml send-comment [report.md](http://report.md/)
<1s
+ cml send-comment  report.md
{"level":"error","message":"Cannot read properties of undefined (reading 'constructor')","stack":"TypeError: Cannot read properties of undefined (reading 'constructor')\n    
at BitbucketCloud.request (/usr/lib/node_modules/@dvcorg/cml/src/drivers/bitbucket_cloud.js:361:14)\n    
at BitbucketCloud.paginatedRequest (/usr/lib/node_modules/@dvcorg/cml/src/drivers/bitbucket_cloud.js:339:41)\n    
at BitbucketCloud.commitPrs (/usr/lib/node_modules/@dvcorg/cml/src/drivers/bitbucket_cloud.js:77:28)\n   
at CML.commentCreate (/usr/lib/node_modules/@dvcorg/cml/src/cml.js:144:41)\n    
at async Object.exports.handler (/usr/lib/node_modules/@dvcorg/cml/bin/cml/send-comment.js:13:15)"}
```
I have installed the Pull Request Commit Links application as described in the docs by clicking on the 'Go to pull request' link and everything else seems to be working but I can't work out what the is causing this error.

I have also tried adding --pr to the send-comment command with no noticeable effect.

Is there something obvious (or otherwise) that I am missing?

Thanks

**Comment by dacbd** [comment]
@pdmct I think I see an error in our code.

**Comment by 0x2b3bfa0** [comment]
@pdmct, fixed with [`v0.15.0`](https://github.com/iterative/cml/releases/tag/v0.15.0), sorry for the inconvenience!
